README.md

```diff
@@ -139,10 +142,70 @@ mysql --user=ci_user --password=ci_pw
 Use `^C` to stop the running MySQL container,
 or -- if you used the detached flag `-d` with `docker-compose up` -- use `docker-compose down`.
 
-Data in the MySQL database will persist after the container is stopped. 
-The MySQL data is stored in a volume mapped to the `.data/` directory in the project. 
+Data in the MySQL database will persist after the container is stopped.
+The MySQL data is stored in a volume mapped to the `.data/` directory in the project.
```

Comment:
Please excuse this whitespace change.  I don't think it's significant to Markdown.

My git client didn't tell me about this difference, but GitHub notices it.  I tried to undo the change and commit it,so it wouldn't show up as a change in this PR, but my git client won't recognize my undoing.